client/src/components/LoginSession/tests/LoginSession-test.js

```diff
@@ -1,37 +1,39 @@
 /* global jest, jasmine, describe, it, expect, beforeEach, Event */
 
 import React from 'react';
-import LoginSession from '../LoginSession';
+import { Component } from '../LoginSession';
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import backend from 'lib/Backend';
 import MockDate from 'mockdate';
 
+// This is to simulate clicking the green 'Remove login session' button on the confirm modal
 jest.mock('@silverstripe/reactstrap-confirm', () => jest.fn().mockImplementation(
     () => Promise.resolve(true)
 ));
 
 Enzyme.configure({ adapter: new Adapter() });
 
-describe('LoginSession', () => {
-    let props = null;
+window.ss.config = {
+    SecurityID: 1234567890
+};
 
-    beforeEach(() => {
-        async function endpointFetcher() {
-            return {
-                error: false,
-                success: true
-            };
-        }
-        backend.createEndpointFetcher = jest.fn().mockImplementation(() => endpointFetcher);
+const setEndpointFetcher = (success, message) => {
+    const endpointFetcher = async () => ({
+        error: false,
+        success,
+        message
+    });
+    backend.createEndpointFetcher = () => endpointFetcher;
```

Comment:
Did you try `jest.mock` to mock `backend` like you did for `@silverstripe/reactstrap-confirm`. My understanding is that's the recommended/standard way of doing it.